charts/chart-museum/templates/deployment.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.global.chartMuseumEnabled }}
```

Comment:
I imagine that the namespace, securityContext, image + tag, limits+requests, and storage should all be configurable? Any other thoughts?

Reply:
Shouldn't the namespace be the same as the astronomer deployment? IE: if this is a component of astronomer, shouldn't it live with astronomer?

Reply:
Makes sense to me. I guess functionally it's similar to the `registry` which is also located there? I am not sure I fully grok when something is "astronomer" vs not - but that can be a convo for another day. Will move this

Reply:
Also yes on configurable limits, requests, and storage.

Reply:
looks like the houston stuff just didn't have a namespace - so I removed it